src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -175,7 +175,7 @@ public function cookie($key = null, $default = null)
     /**
      * Get an array of all of the files on the request.
      *
-     * @return array
+     * @return array<int, \Illuminate\Http\UploadedFile|\Illuminate\Http\UploadedFile[]>
```

Comment:
I think most codebases don't take an array containing another array of files into consideration, but it is possible according tot he code. `convertUploadedFiles` recursively maps the array to files or if the array contains an array, to an array of files (inside the array of files) resulting in nested arrays of files.

I guess not flat mapping this is on purpose (would be a breaking change now anyway), so the docblocks reflect the possibility of an array of items of which each item is either a file or an array of files.